tests/operators/test_cartesian_sampling_op.py

```diff
@@ -31,35 +32,40 @@ def test_cart_sampling_op_data_match():
     sx = 'uf'
     sy = 'uf'
     sz = 'uf'
-    ktraj = create_traj(k_shape, nkx, nky, nkz, sx, sy, sz)
+    trajectory = create_traj(k_shape, nkx, nky, nkz, sx, sy, sz)
 
     # Create matching data
     random_generator = RandomGenerator(seed=0)
     kdata = random_generator.complex64_tensor(size=k_shape)
 
     # Create sampling operator
     encoding_shape = SpatialDimension(k_shape[-3], k_shape[-2], k_shape[-1])
-    SOp = CartesianSamplingOp(encoding_shape=encoding_shape, traj=ktraj)
+    sampling_op = CartesianSamplingOp(encoding_shape=encoding_shape, traj=trajectory)
```

Comment:
maybe `cart_sampling_op`?

Reply:
if we wanna rename it, I would suggest `carteasian_sampling_op`, but since we are in `test_cartesian_sampling_op.py` and only deal with cartesian data, I think we can stick with just `sampling_op` here